pmd-core/src/main/java/net/sourceforge/pmd/cpd/TokenEntry.java

```diff
@@ -48,6 +50,25 @@ public TokenEntry(String image, String tokenSrcID, int beginLine) {
         setImage(image);
         this.tokenSrcID = tokenSrcID;
         this.beginLine = beginLine;
+        this.beginColumn = -1;
```

Comment:
Please call here the other constructor via `this(image, tokenSrcID, beginLine, -1, -1)`. That way, we don't need to duplicate the logic...